src/openrct2/rct2/S6Importer.cpp

```diff
@@ -410,7 +410,7 @@ namespace RCT2
             // rct1_water_colour
             // Pad01358842
             ImportResearchList(gameState);
-            gMapBaseZ = _s6.MapBaseZ;
+            gameState.MapBaseZ = _s6.MapBaseZ;
```

Comment:
What even is this variable for? It looks to always be set to 7 except on s6 import where it might be something else. @Gymnasiast any info on this?

Reply:
It controls which land height counts as 0 metres/feet. This can be seen when viewing height labels and watching the altitude graph for a roller coaster. It appears to be mostly cosmetic.

Reply:
Can't we just change it to a hardcoded constant of 7 since nothing in the ui lets you modify it. Unsure if existing s6 files have this changed.

Reply:
I would be reluctant to do so, it seems like a feature that could be nice for scenarios are set at a particularly low or high altitude. It’s hardly an intrusive thing either. I would prefer exposing the option instead.

OTOH, I’m not particularly wedded to it, so if other team members agree with removing it, I’m not going to object.

Reply:
okay created an issue for it.